pkg/roachpb/api.proto

```diff
@@ -1424,6 +1424,8 @@ message ExternalStorage {
     string billing_project = 4;
 
     string credentials = 5;
+
+    string service_account = 6;
```

Comment:
depending on how we address the comment on the URI param name we should change this accordingly. A comment might still be nice

Reply:
added comment